app/lib/cds_importer/entity_mapper/base_mapper.rb

```diff
@@ -108,42 +108,21 @@ def parse
         if mapping_path.present?
           expanded.select! { |values| values.dig(*mapping_path.split(PATH_SEPARATOR)).present? }
         end
-        expanded.map { |values| create_instance(values) }
+        expanded.map(&method(:build_instance))
```

Comment:
I guess values are passed implicitly here 🤔

Reply:
@alexdesi Yeh, works the same way as symbol to proc - symbol to proc in effect recognises its a proc and just calls `#method` on the object handed in - where as this passes in the method from the calling class

Its the equivalent of javascript's ability to use functions like variables

```javascript
function sayHello() {
  console.log("Hello world!")
}

setTimeout(sayHello, 5000)
```